server/tools/hatohol-db-initiator.in

```diff
@@ -132,6 +132,11 @@ def parse_default_conf():
 
 def get_default_conf_item(item_name):
     config = parse_default_conf()
+    if config is None:
+       global not_found_config_file
+       not_found_config_file = True
+       return "<Unknown>"
```

Comment:
> Do you think these should have default values ?

I think we don't need to provide default values for them when the "-c" options is used.
But ... I have no certain rationale for it.